### ISSUE TITLE
[SF][0.31.19][UI5-Dialog][BUG][Urgent - Expected WithIn Two week] accessibleName and aria-describedby does not work in UI5 Dialog

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#4004]
**Dear colleagues,**
I am from SuccessFactors **Web Experience Shanghai team.**
we use **NVDA** to test a11y issues in our project

**The version of ui5-webcomponent is 0.31.19**

**Reproduce Step:**
1. open [https://github.wdf.sap.corp/pages/xweb/common-react/?path=/story/react-components-first-time-guidance--default](https://github.wdf.sap.corp/pages/xweb/common-react/?path=/story/react-components-first-time-guidance--default)
2. click "show the first normal slide"

![Screen Shot 2021-09-27 at 4 29 50 PM](https://user-images.githubusercontent.com/78520316/134872945-49e076c1-b13a-4258-a692-247fb1e45a80.png)

![Screen Shot 2021-09-27 at 2 20 36 PM](https://user-images.githubusercontent.com/78520316/134873001-0c8467c2-c89a-4f2a-8f9a-010f52418f71.png)

**Current Behavior:**
1. The accessible-name is "test accessibleName attribute for dialog", but this text is not announced by screen reader
2. The content belong to aria-describedby is "first slide", but this text is not announced by screen reader

**Expected Behavior:**
1. The accessible-name can be announced by screenreader correctly
2. The content belong to aria-describedby "first slide" can be announced by screenreader correctly

**- Organization: SF**

**Comment by Todor-ads** [comment]
Hi @SAP/ui5-webcomponents-topic-rd,

The Dialog component is part of your topic.

Best regards,
Todor

**Comment by dimovpetar** [comment]
Hello @sfsf-xweb-sh ,

The issue with `accessibleName` property of the Dialog has been fixed with https://github.com/SAP/ui5-webcomponents/pull/3936. Please await the next patch for release-0.31. @ilhan007 can you cooperate on this?
The usage of `aria-describedby` is not supported. We set specific aria attributes on the section inside the shadow DOM of the Dialog - this is what gets announced and the reason why setting it externally has no effect. In your case you should use the `accessibleName` for this one as well. However, if you use JAWS for Screen Reader, the content is read out without setting any attributes.

Best regards,
Petar

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh 

as @dimovpetar said the issue has been resolved with #3936
The change will be available with 0.31.20. For now I am keeping the issue open,
 so you can confirm and close it yourself afterwards.

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh the 0.31.20 is released and accessibleName has been added to the Dialog.

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh it has been 7 days since the patch release, we are closing the issue now. 
If you still face issues, let us know.
